CHANGELOG.md

```diff
@@ -5,6 +5,105 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.11.0] - 2023-01-11
+
+This v0.11.0 release of the Hubble CLI adds support for features added in
+Cilium v1.13: Hubble now has visibility into Cilium's SockLB,
+meaning it is possible to observe service address translations performed
+by Cilium on the socket level (#816). Hubble CLI v0.11 also supports the newly
+introduced Cilium v1.13 flow filters for workload and trace ID (#794, #795).
+
+This release also removes the old and deprecated JSON formatter and now always
+uses to the more flexible proto3-based `jsonpb` output when JSON is selected
+as the output format. This is a potentially breaking change and requires that
+e.g. `jq` queries of the form `hubble observe -o json | jq .source` are
+rewritten as `hubble observe -o json | jq .flow.source` (#826).
```

Comment:
Could we also list #826 under a `**Breaking Changes**` section below?

Reply:
Sure! I've updated the PR to mirror the breaking changes sections we had in previous releases.